conda-forge.yml

```diff
@@ -1,3 +1,6 @@
+build_platform:
+  linux_aarch64: linux_aarch64
+  osx_arm64: osx_64
```

Comment:
@synapticarbors would you like to look this change? I’m not sure whether could build for M1 Mac.